### DISCUSSION TITLE
Traefik example

### DISCUSSION THREAD

**Discussion by tugdualenligne** [ggogel/seafile-containerized#18] [Q&A] [answered]
Hi there, I enjoyed your docker-compose
Would you mind providing a Traefik example as you did for Nginx? It will help a lot as I'm a bit lost (I got to have a Pydio instance running behind Traefik, but with only one port, here I'm lost with the different ports required for Seafile, Seahub, etc.)
Many thanks in advance

**Comment by ggogel** [comment, marked as answer]
Hey, you only need to point traefik to port 80 of seafile-caddy, it'll do the rest.

**Reply by tugdualenligne** [reply]
Thanks. I got it to work but it seems to create an issue with the db. I can connect to Seafile and view the apps pages but I have some errors in them and no files are appearing. I get in the docker console this error message "[Warning] Aborted connection 3 to db: 'unconnected' user: 'root' host: '172.19.0.5' (Got an error reading communication packets)"
Any idea?

At the seafile-caddy level, I inserted the following:
    labels:
      - traefik.enable=true
      - traefik.http.services.seafile-caddy.loadbalancer.server.scheme=http 
      - traefik.http.routers.seafile-caddy.rule=Host(`${PUBLIC_FQDN}`)
      - traefik.http.routers.seafile-caddy.entrypoints=websecure
      - traefik.http.routers.seafile-caddy.tls=true
      - traefik.http.routers.seafile-caddy.tls.certresolver=leresolver
      - traefik.http.routers.seafile-caddy-secure.service=seafile-caddy

Many thanks in advance
